src/tests/roc_lib/test_sender_receiver.cpp

```diff
@@ -223,6 +224,7 @@ TEST_GROUP(sender_receiver) {
         receiver_conf.n_repair_packets = n_repair_packets;
         receiver_conf.latency = packet_len * 20;
         receiver_conf.timeout = packet_len * 300;
+        receiver_conf.sample_rate = pipeline::DefaultSampleRate;
```

Comment:
Since we have a default back, this line can be removed.

Reply:
Fixed